browser/app/js/objects/ObjectsBulkActions.js

```diff
@@ -112,6 +102,16 @@ export class ObjectsBulkActions extends React.Component {
       )
     }
 
+    let downloadLabel
+
+    if (checkedObjects.length === 1 && !checkedObjects[0].endsWith("/")) {
+      downloadLabel = "Download Object"
```

Comment:
Another minor nitpick, however do you think we should make the word `Object` start with a lowercase letter here (So that this option is `Download object` instead of `Download Object`) as all the other options don't have any capital letters after the first letter (Such as `Delete selected` - s is lowercase, `Download as zip`, `Download all as zip` - a and z are lowercase)?